src/games/strategy/triplea/ui/screen/IDrawable.java

```diff
@@ -128,26 +128,29 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
         return;
       }
     }
-    final Rectangle territoryBounds = mapData.getBoundingRect(territory);
+
+    // get font metrics
```

Comment:
unnecessary comment. We have the word 'getFontMetrics()' three lines later, comment is repeating the code.